docs/components/static-index.jsx

```diff
@@ -15,7 +15,7 @@ class Index extends React.Component {
           <meta charSet="utf-8" />
           <meta httpEquiv="X-UA-Compatible" content="IE=edge,chrome=1" />
           <title>Victory</title>
-          <meta name="description" content="" />
+          <meta name="description" content="Our latest OSS endeavor: An ecosystem of modular data visualization components for React.js" />
```

Comment:
Let's avoid using the abbreviation "OSS".

Perhaps:
> An open source ecosystem of modular data visualization components for React.js

@elizagrace this is for you!